Update README.md

ploetzly opened the pull request:
Major update to reflect current hyperchains documentation. Added usage and troubleshooting sections, updated most other sections.

ploetzly committed:
Update README.md

major update to reflect current hyperchains documentation. added usage and troubleshooting sections, updated most other sections.

dincho commented on README.md:
```diff
@@ -5,65 +5,162 @@ The goal of this project is to ease the process of bootstrapping an Aeternity hy
+The toolkit provides automated workflows for:
+1. Generating and configuring the Hyperchain economy (initial accounts, balances, and contracts)
+2. Setting up node configuration files
+3. Deploying and initializing required smart contracts
```
This sounds a bit wrong to me as the tool does not do any deployment at all. It generates the corresponding configuration files that the node then uses to auto-deploy the contracts under the hood on genesis block.

ploetzly replied:
would this be better:
## Structure
The toolkit provides automated workflows for:
- Generating and configuring the Hyperchain economy configuration (initial accounts and balances)
- Preparing contract configurations for genesis block deployment
- Setting up node configuration files and validator accounts
- Generating the necessary configuration files for pinning operations

Each component is configured through YAML files that will be used by the node during initialization. The node will then handle actual contract deployment and account setup during the genesis block creation.

klercker replied:
I would perhaps take out the word "files" in the last bullet, since it's actually config within the single configuration file, not a separate one. This to avoid confusion.

ploetzly replied:
revision:
## Structure
The toolkit provides automated workflows for:
1. Generating Hyperchain economy configuration parameters (initial accounts and balances)
2. Preparing contract configurations for genesis block deployment
3. Creating validator account configurations
4. Preparing pinning operation parameters

Each component generates configuration output that the Aeternity node will use during initialization and genesis block creation.

ploetzly replied:
revision:
## Structure
The toolkit provides automated workflows for:
1. Generating Hyperchain economy configuration parameters (initial accounts and balances)
2. Preparing contract configurations for genesis block deployment
3. Creating validator account configurations
4. Preparing pinning operation parameters

Each component generates configuration output that the Aeternity node will use during initialization and genesis block creation.

dincho commented on README.md:
```diff
@@ -5,65 +5,162 @@ The goal of this project is to ease the process of bootstrapping an Aeternity hy
 ## Prerequisites/assumptions
 - You have a somewhat solid understanding of how blockchains work and [æternity](https://github.com/aeternity/aeternity) in particular.
-- You are comfortable with the command line, you are a semi-competent unix administrator.
+- A running Aeternity node (v6.7.0 or later)
```
This tool doesn't actually needs a node, it outputs the artefacts that the node needs, so more or less it's inverse dependancy

dincho commented on README.md:
```diff
@@ -5,65 +5,162 @@ The goal of this project is to ease the process of bootstrapping an Aeternity hy
+- [Node.js](https://nodejs.org/en/download) installed
+- [Git](https://git-scm.com/downloads) installed
+
+## Technical Requirements:
```
Those are node requirements, this tools doesn't need any of this. This tool requirements are relatively basic that doesn't need any docs.

klercker replied:
@dincho is right. The tool does not do these things and does not need a node to be used. To be *useful* though, you need an installed node where to put the output of the tool (as per the core docs).

ploetzly replied:
revision:
## Prerequisites/assumptions
For using the toolkit:
- [Node.js](https://nodejs.org/en/download) installed
- [Git](https://git-scm.com/downloads) installed
- Basic command line familiarity

For using the generated configurations:
- A running Aeternity node (v6.7.0 or later) where you'll apply the generated configurations
- Experience managing blockchain nodes
- Access to sufficient funds for pinning operations
- Understanding of node validator operations

Note: While this tool can be run without an Aeternity node installed, its output is designed to be used with a node. The configurations generated are only useful when applied to an actual node deployment.

ploetzly replied:
revision:
## Prerequisites/assumptions
For using this tool:
- Basic command line familiarity
- [Node.js](https://nodejs.org/en/download) installed
- [Git](https://git-scm.com/downloads) installed

For using the generated output:
- An Aeternity node (v6.7.0 or later) install location where you will place the generated configurations

ploetzly committed:
Update README.md

edited certain statements that weren't accurate.
NOTE: resources/links at bottom should be updated as soon as Gitbook is live/more current FAQs are made (unless this repo is deprecated).
